src/main/resources/fabric.mod.json

```diff
@@ -29,7 +29,7 @@
     "fabric-lifecycle-events-v1": "*",
     "fabric-rendering-v1": "*",
     "fabric-resource-loader-v0": ">=0.4.7",
-    "minecraft": ">=1.17",
+    "minecraft": "1.17.x",
```

Comment:
This is useless as it works as-is on 1.18.

I refuse to constrain too strictly Minecraft versions as it ends up being more of a pain for everyone.

Reply:
Agreed - constraints should only be added if there is breaking behaviour.